ht/cogs/reference.py

```diff
@@ -168,6 +168,14 @@ async def familycrest(self, ctx):
 	async def fridge(self, ctx):
 		with open("media/prose/fridgetesting.md", "r") as file:
 			await ctx.send(file.read())
+			
+	@commands.command(
+		help = "Shows a short blurb about false quartering.",
+		aliases = ("fq", "false-quartering")
```

Comment:
```suggestion
		aliases = ("fq", )
```

None of the other commands have aliases with kebab case versions so for consistency I’d rather avoid this.